### DISCUSSION TITLE
Change month programatically on button tap

### DISCUSSION THREAD

**Discussion by rahul2681993** [kvyatkovskys/KVKCalendar#115] [Q&A] [answered]
Hello Kviatkovskii,

I want to change the month with the tap of a button but I didn't find any method for it.  Could you please help with that?

**Example:**  On the button tap, I want to change `December` to `January`and vice versa.

Best regards,
~Rahul

**Comment by kvyatkovskys** [comment, marked as answer]
Hello, 🤔 
You can subscribe on this method `didSelectDate(_ date: Date?, type: CalendarType, frame: CGRect?)` then check the date and type.... and then to change the date `calendarView.set(.month, date: Date)`

**Comment by rahul2681993** [comment]
Thanks for your reply, I got the answer right after dropping that question.


Best regards,
Rahul
